django_socio_grpc/filters.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.filters import OrderingFilter as RestOrderingFilter
+
+
+class OrderingFilter(RestOrderingFilter):
+    def get_ordering(self, request, queryset, view):
+        """
+        Allow ordering with direct array value as grpc can pass array directly for ordering
+        where REST Ordering is set by a comma delimited ?ordering=... query parameter.
+
+        The `ordering` query parameter can be overridden by setting
+        the `ordering_param` value on the OrderingFilter or by
+        specifying an `ORDERING_PARAM` value in the API settings.
+
+        Direct link of DRF OrderingFilter: https://github.com/encode/django-rest-framework/blob/master/rest_framework/filters.py
+        """
+        params = request.query_params.get(self.ordering_param)
+        if params:
+            if isinstance(params, str):
```

Comment:
As the behavior is grpc specific is it not better to check for a list instead of an str and to process standard http request params with standard tooling and avoid manually splitting stuff

Reply:
It can be but it will be doing the work twice. Let me explain.
As query params are string. DRF split the string to an array. So if we check if it's an array we will need to join it to transform it into a string to send it to the parent class that will then transform back into an array.

So both solution have pro and con. Honestly, If you think that have string transformation twice is better than maintain overrided code I can change it. It is not a big con for me.

Reply:
I would prefer to join the list even if after that DRF is going to re-split it. We will have less code to maintain and stick closer to the native DRF behavior

Reply:
As discussed in MP, I am not fan of modifying directly the query_params. Can we merge this PR ?